Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:14.15.1
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY ["package.json", "/usr/src/app"]
+
+RUN npm i
+
+RUN npm install -g nodemon
```

Comment:
Move nodemon to dev deps, so you won't need to have it installed globally.

Reply:
yea, i forgot remove this line. it is already in deps